### PR TITLE
[TF Hub][TF FE][GA] Establish regular validation for all TF Hub models

### PR DESCRIPTION
**Pull request by rkazants**
**Details:** Establish regular validation for all TF Hub models in GitHub Actions

**Ticket:** 117711

### CODE REVIEW

File: .github/workflows/linux.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 name: Tests on Linux (Ubuntu 22.04, Python 3.11)
 on:
+  schedule:
+    # - cron: '0 20/72 * * *'
+    - cron: '10 * * * *'
```
Comment (rkazants) [inline .github/workflows/linux.yml:5]: Now it is just only for experiment to have every 10 minutes on the precommit.
If the experiment is successful, I will switch to every 3 days validation on full scope.

File: tests/model_hub_tests/tf_hub_tests/nightly_models_tmp [hunk 2]
```diff
@@ -0,0 +1,2 @@
+vision/embedder/fungi_V2,https://tfhub.dev/svampeatlas/vision/embedder/fungi_V2/1?tf-hub-format=compressed
```
Comment (rkazants) [inline tests/model_hub_tests/tf_hub_tests/nightly_models_tmp:1]: Temporal file.

**Review by akashchi — APPROVED** [review]

File: .github/workflows/linux.yml [hunk 3]
```diff
@@ -674,8 +677,9 @@ jobs:
 
           source ${{ env.INSTALL_DIR }}/setupvars.sh
 
-          python3 -m pytest ${{ env.MODEL_HUB_TESTS_INSTALL_DIR }}/tf_hub_tests/ -m precommit --junitxml=${{ env.INSTALL_TEST_DIR }}/TEST-tf_hub_tf_fe.xml
+          python3 -m pytest ${{ env.MODEL_HUB_TESTS_INSTALL_DIR }}/tf_hub_tests/ -m ${{ env.TYPE }} --junitxml=${{ env.INSTALL_TEST_DIR }}/TEST-tf_hub_tf_fe.xml --html TEST-tf_hub_tf_fe.html --self-contained-html
         env:
+          TYPE: ${{ github.event_name == 'schedule' && 'nightly' || 'precommit'}}
```
Comment (akladiev) [inline .github/workflows/linux.yml:682]: How long does it take to execute a full nightly scope?

Comment (rkazants) [reply .github/workflows/linux.yml:682]: Discussed offline

**Pull request merged**